app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/dataprodukt/Dataprodukt.kt

```diff
@@ -26,12 +27,12 @@ object Dataprodukt {
             launch {
                 database.await()
```

Comment:
denne launch-en kan fjernes. tok den bare med for å være helt sikker på at databasen initialiseres.